script/build.js

```diff
@@ -22,8 +22,22 @@ const webpack = require('metalsmith-webpack');
 const webpackConfigGenerator = require('../config/webpack.config');
 const webpackDevServer = require('metalsmith-webpack-dev-server');
 
+const fs = require('fs');
+
 const sourceDir = '../content/pages';
 
+// Make sure git pre-commit hooks are installed
+['pre-commit'].forEach(hook => {
+  const src = `hooks/${hook}`;
+  const dest = `.git/hooks/${hook}`;
+  if (fs.existsSync(src)) {
+    if (!fs.existsSync(dest)) {
```

Comment:
There are tools to manage what hooks are installed, but I like the simplicity of this.